bundles/org.openhab.ui.dashboard/README.md

```diff
@@ -0,0 +1,34 @@
+# Dashboard tiles
+
+OpenHAB dashboard is landing page for the user where all openHAB UI's can be found. Dashboard support also links to external services. Links can be added to dashboard by ```conf/services/dashboard.cfg``` configuration file.
+
+## Link Configuration
+
+| Parameter name  | Type    | Description                                                                             |
+|-----------------|---------|-----------------------------------------------------------------------------------------|
+| link-nameX      | String  | Name which is shown in the openHAB dashboard.                                           |
+| link-urlX       | String  | URL to external service.                                                                |
+| link-overlayX   | String  | Image overlay icon. Supported values are empty (no icon), "html5", "android" or "apple" |
+| link-imageurlX  | String  | URL to image.                                                                           |
+
+Where X is link unique number (see examples). All configuration parameters need to start with ```org.openhab.ui.dashboard:``` prefix.
```

Comment:
We should change the service pid from `org.openhab.ui.dashboard` to `org.openhab.dashboard`. With that change, the pid is derived from the filename and you can leave out the prefix in the file:

```
link-name1=openHAB Log Viewer
link-url1=http://localhost:9001
link-overlay1=
link-imageurl1=http://localhost:8080/static/image.png

```
Would be much nicer!